retrofit/src/main/java/retrofit2/Response.java

```diff
@@ -147,12 +147,22 @@ public boolean isSuccessful() {
     return rawResponse.isSuccessful();
   }
 
-  /** The deserialized response body of a {@linkplain #isSuccessful() successful} response. */
+  /**
+   * The deserialized response body of a {@linkplain #isSuccessful() successful} response.
+   *
+   * <p>If the body is of type {@link ResponseBody}, {@link ResponseBody#close()} must always be
+   * called to avoid leaking resources
```

Comment:
This is only true of `@Streaming` endpoints. I think this doc needs to live only on the `@Streaming` annotation and not here.

The one down below is still useful, however.

Reply:
Thanks, that makes sense. I'll try to update it tonight.